charts/renku/templates/configmap.yaml

```diff
@@ -53,7 +53,7 @@ data:
         -d  '{"username": "demo",
         "email": "demo@datascience.ch",
         "name": "John Doe",
-        "extern_uid": "demo",
+        "extern_uid": "e144b235-793b-4e2e-bb1f-1f8baccc321f",
```

Comment:
this is always the same value?

Reply:
yes, it's hard-coded in the realm template